scripts/release_certification.py

```
@@ -0,0 +1,899 @@
+    if set(profiles["pr"]) != required_pr:
+        raise CertificationError("PR profile must contain exactly the five selective behaviors")
+    required_full = required_pr | {
+        "pre_v8_hard_cut_source",
+        "protocol_installer_boundaries",
+    }
+    if not required_full.issubset(profiles["full"]):
+        raise CertificationError("full profile omits a required behavior class")
+
+    platforms = policy["profile_platform_sets"]
+    if not isinstance(platforms, dict) or set(platforms) != set(profiles):
+        raise CertificationError("profile_platform_sets must exactly cover profiles")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)